openshift/e2e-tests-openshift.sh

```diff
@@ -196,7 +197,21 @@ EOF
 }
 
 function deploy_serverless_operator(){
-  oc apply -f openshift/serverless/operator-install.yaml
+  git clone https://github.com/openshift-knative/serverless-operator.git /tmp/serverless-operator
+  /tmp/serverless-operator/hack/catalog.sh | oc apply -n $CS_NS -f -
+  cat <<-EOF | kubectl apply -f -
```

Comment:
Not sure you need this `-` in `<<-EOF`. I always use it without.

Reply:
I borrowed this as is from https://github.com/openshift-knative/serverless-operator/blob/master/README.md#create-a-subscription from @jcrossley3 , may be he has something to add here.